R/summary_functions.R

```diff
@@ -191,6 +264,58 @@ print.coverages <- function(x, ...) {
   invisible(x)
 }
 
+#' Print the branch coverage of a coverage object
+#'
+#' @param x the coverage object to be printed
+#' @param group whether to group coverage by filename or functions
+#' @param ... additional arguments ignored
+#' @return The coverage object (invisibly).
+#' @export
+print_branch_coverage <- function(x, group = c("filename", "functions")) {
```

Comment:
The same here, can there be an abstraction between print_coverage and print_branch_coverage?

Reply:
Possibly. I will try.

Reply:
I looked at it myself and it does not look like. Keep it like it is.

Reply:
Ok!